scriptworker/task.py

```diff
@@ -448,6 +465,12 @@ def prepare_to_run_task(context, claim_task):
         except taskcluster.exceptions.TaskclusterRestFailure as exc:
             if exc.status_code == 409:
                 log.debug("409: not reclaiming task.")
+                if context.proc and task == context.task:
+                    await kill_task(
+                        context,
+                        "Killing task after receiving 409 status in reclaim_task",
```

Comment:
409 means we can't reclaim the task because it's no longer ours to reclaim. Aiui, that includes

- superseded (taken by another worker)
- cancelled
- deadline-exceeded
- marked as completed/exception somehow (task finished in another async coroutine?)